src/main/java/datastructs/NumericSample.java

```diff
@@ -93,7 +99,7 @@ public final Statistics getStatistics(){
 	public final void add(Double value){
 
 		this.data_.add(value);
-		this.stats_.is_valid = false;
+		this.falsifyCalculations();
```

Comment:
could explain a little bit about `falsifyCalculations()`?

Reply:
check what the function does.....it simply sets to false the flags ```this.stats_.is_valid```
and the ```is_sorted```... since we want to change two variables when either one is changings I wrapped it in a function that's all